.github/workflows-src/steps.lib.yml

```diff
@@ -102,7 +102,11 @@ name: "Critical Integration Tests"
 shell: bash
```

Comment:
Doesn't look like this PR needs the state tool binaries rebuilt, please remove them so we don't needlessly balloon our repo size.

Reply:
This is necessary to bring in the _prepare cmd changes. I opted to do it in this PR since this PR introduces the calls to that cmd.